rpcs3/Emu/SysCalls/Modules/sys_mempool.cpp

```diff
@@ -1,40 +1,129 @@
 #include "stdafx.h"
 #include "Emu/Memory/Memory.h"
 #include "Emu/System.h"
+#include "Emu/IdManager.h"
 #include "Emu/SysCalls/Modules.h"
 
 #include "sysPrxForUser.h"
 
 extern Module sysPrxForUser;
 
+using sys_mempool_t = u32;
+
+struct memory_pool_t
```

Comment:
I don't see the point of exposing this struct in the header file, it is not supposed to be used outside this compilation unit.